phone_bank_result.md

```diff
@@ -0,0 +1,804 @@
+    * [Scenario: Retrieving a collection of Phonebank result resources for a phone bank (GET)](#scenario-retrieving-a-collection-of-phone-bank-result-for-a-phone-bank-get)
+    * [Scenario: Retrieving an individual Phonebank result resource (GET)](#scenario-retrieving-an-individual-phone-bank-result-get)
+    * [Scenario: Retrieving Answers for an individual Phonebank result resource (GET)](#scenario-retrieving-answers-for-an-individual-phone-bank-result-get)
+    * [Scenario: Retrieving Taggings for an individual Phonebank result resource (GET)](#scenario-retrieving-taggings-for-an-individual-phone-bank-result-get)
+    * [Scenario: Creating a new Phonebank result (POST)](#scenario-creating-a-new-phone-bank-result-post)
+    * [Scenario: Modifying a Phonebank result (PUT)](#scenario-modifying-a-phone-bank-result-put)
+    * [Scenario: Deleting a Phonebank result (DELETE)](#scenario-deleting-a-phone-bank-result-delete)
+
+
+{% include endpoints_and_url_structures.md %}
+
+The link relation label for an Phone bank result resource is ```osdi:phone-bank-result``` for a single resource or ```osdi:phone-bank-results``` for a collection of resources.
+
+_[Back to top...](#)_
+
+
+## Fields
+
+{% include fields_intro.md %}
+
+{% include global_fields.md %}
+
+_[Back to top...](#)_
+
+
+### Phone bank result Fields
+
+| Name          | Type      | Description
+|-----------    |-----------|--------------
+|origin_system	  |string    |A human readable identifier of the system where this Phonebank result was created. (ex: "OSDI System")
+|call_date		  |string	 |The date and time the call was made.
+|caller_id        |string    |The phone number that is displayed as the caller-id for this call
+|phone_number     |string    |The phone number that was called
+|duration         |string    |The length of the phone call in seconds. Calls that do not connect will have zero in this field.
+|disposition      |enum      |Disposition of the phone call. Possible values are "answer". "busy", "do-not-call", "meaningful interaction", "machine" etc..
+
+_[Back to top...](#)_
+
+
+
+### Links
+
+{% include links_intro.md %}
+
+|Name          	|Type		|Description
+|-----------    |-----------|--------------
+|self			|[Phonebank Result*](phone-bank-result.html)	|A self-referential link to the Phonebank result.
+|caller         |[Person*](people.html)         |A link to a single Person resource representing the person who made the call
+|target		    |[Person*](people.html)  |A link to a single Person resource representing the person who was contacted.
+|answers        |[Answers[]*](answers.html) | A link to the collection of Answers to [Questions](questions.html) posed during the Phonebank.
+|attendances    |[Attendances[]*](attendances.html) | A link to the collection of Attendances to [Events](events.html) posed during the Phonebank.
```

Comment:
let's clarify the description a bit maybe -- I get what it's going for, just could be more clear